.github/workflows/cff.yml

```diff
@@ -0,0 +1,22 @@
+name: validate-citation-file
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+      
+  workflow_dispatch:
+
+jobs:
+  validate-citation-cff:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I saw this also in the examples of the validation action, but I am wondering why it is needed? We do check out the repo before the validation step, is that action querying more stuff from our repo via the API?

Reply:
Looking at the action code (https://github.com/dieghernan/cff-validator/blob/main/action.yml), I don't see where `GITHUB_PAT` is used, but I guess it wouldn't be in the examples if it wasn't necessary... I'm just usually a bit hesitant to use secrets in the CI :)

Reply:
I was also not sure about the env variable. It seems that the action is based on R and that authentication is needed for set-up or something ...